lib/google-apis/auth-client.js

```diff
@@ -24,7 +24,7 @@ var isString = lodash.isString;
 var SCOPES = ['https://www.googleapis.com/auth/cloud-platform'];
 
 /* @const {String} Base Error Reporting API */
-var API = 'https://clouderrorreporting.googleapis.com/v1beta1/projects'
+var API = 'https://clouderrorreporting.googleapis.com/v1beta1/projects';
```

Comment:
Is `jshint` running on each change? I would have thought it caught things like this